src/main/java/com/google/sps/JobModel.java

```diff
@@ -68,15 +76,14 @@ public JobModel(String projectId, Job job, Dataflow dataflowService) throws IOEx
       
       SdkVersion sdkVersion = job.getJobMetadata().getSdkVersion();
       if (sdkVersion != null) {
-          sdk = sdkVersion.getVersionDisplayName();
+          sdk = sdkVersion.getVersion();
           sdkSupportStatus =  sdkVersion.getSdkSupportStatus();
       }
 
       region = job.getLocation();
 
       Environment environment = job.getEnvironment();
-      if (environment != null) {
-          workerLocation = environment.getWorkerRegion();
+      if (environment != null) {;
 
```

Comment:
```suggestion
```
Remove extra space.

Reply:
resolved in https://github.com/googleinterns/step229-2020/pull/15/commits/d93c9abd9592079a818b7ad7396acdd078cc1aab